pkg/metrics/timer.go

```diff
@@ -0,0 +1,42 @@
+package metrics
+
+import (
+	"time"
+
+	"github.com/sirupsen/logrus"
+)
+
+// Timer is the struct that keeps track of each of the sections.
+type Timer struct {
+	startTime    time.Time
+	totalTime    time.Time
+	listOfStages []string
+	listOfTimes  []time.Duration
+}
+
+var timer Timer
+
+// StartTimer initiailzes the timer object with the current timestamp information.
+func StartTimer() {
+	timer = Timer{}
+	timer.startTime = time.Now().Round(time.Second)
+	timer.totalTime = time.Now().Round(time.Second)
+	timer.listOfStages = []string{"Infrastructure", "API", "Bootstrap Complete", "Bootstrap Destroy", "Cluster operator"}
```

Comment:
operator -> Operator. Maybe we could think of a better name for this. Operators Available  or Cluster Operators Available perhaps.

Reply:
Wasn't sure exactly what to print here so I'm all ears.